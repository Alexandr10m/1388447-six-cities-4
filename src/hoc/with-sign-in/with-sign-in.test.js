import React from "react";
import renderer from "react-test-renderer";
import withSignIn from "./with-sign-in.js";


const MockComponent = () => <form></form>;
const MockComponentWrapped = withSignIn(MockComponent);

describe(`Snapshot of withSignIn`, () => {
  it(`withSignIn should render correctly`, () => {
    const tree = renderer
      .create(
          <MockComponentWrapped/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
