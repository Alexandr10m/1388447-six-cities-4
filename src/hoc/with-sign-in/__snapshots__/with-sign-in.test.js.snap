// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of withSignIn withSignIn should render correctly 1`] = `<form />`;
